src/components/DisplayModal.tsx

```diff
@@ -0,0 +1,130 @@
+import { css } from '@emotion/react';
+import { useEffect } from 'react';
+import { IcDone, IcLeftArrow, IcRightArrow } from '../assets';
+import { imageType } from '../mobile/Display/types/imageType';
+import ModalPortal from './ModalPortal';
+
+interface modalProps {
+  imgId: number;
+  studioImageList?: Array<imageType>;
+  changeClickedImgId: (imgId: number) => void;
+  closeModal: () => void;
+}
+
+const DisplayModal = ({
+  imgId: clickedImgId,
+  studioImageList,
+  changeClickedImgId,
+  closeModal,
+}: modalProps) => {
+  const { imgPath } =
+    studioImageList?.filter((img) => img.imgId === clickedImgId)[0] || {};
+
+  const handleClickBg = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    if (e.target === e.currentTarget) {
+      closeModal();
+    }
+  };
+
+  const handleClickArrow = (arrow: string, imgId: number) => {
+    if (studioImageList) {
+      const isPrevBtnClicked = arrow === 'left';
+      const firstImgId = studioImageList[0].imgId;
+      const lastIdx = studioImageList.length - 1;
+      const lastImgId = studioImageList[lastIdx].imgId;
+
+      if (isPrevBtnClicked) {
+        const changedImgId = imgId === firstImgId ? lastImgId : imgId - 1;
+        changeClickedImgId(changedImgId);
+      } else {
+        const changedImgId = imgId === lastImgId ? firstImgId : imgId + 1;
+        changeClickedImgId(changedImgId);
+      }
+    }
+  };
+
+  useEffect(() => {
+    const scrollY = window.scrollY;
+
+    document.body.style.cssText = `
+      position: fixed; 
+      top: -${scrollY}px;
+      overflow-y: hidden;
+      `;
+
+    return () => {
+      document.body.style.cssText = '';
+      window.scrollTo({
+        top: scrollY,
+        behavior: 'instant',
+      });
+    };
+  }, []);
+
+  return (
+    <ModalPortal>
+      <section css={background} onClick={handleClickBg}>
+        <div css={closeBtn} onClick={closeModal}>
+          <IcDone />
+        </div>
+
+        <article css={imgContainer} onClick={handleClickBg}>
+          <IcLeftArrow
+            css={arrow('left')}
+            onClick={() => handleClickArrow('left', clickedImgId)}
+          />
+          <img src={imgPath} alt="스튜디오 이미지" css={studioImg} />
+          <IcRightArrow
+            css={arrow('right')}
+            onClick={() => handleClickArrow('right', clickedImgId)}
+          />
+        </article>
+      </section>
+    </ModalPortal>
+  );
+};
+
+export default DisplayModal;
+
+const background = css`
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 998;
+
+  width: 100dvw;
+  height: 100dvh;
+
+  /* gray900 opacity 80% */
+  background-color: rgb(31 27 28 / 80%);
+`;
+
+const closeBtn = css`
+  position: absolute;
+  top: calc(100vh / 50.625);
+  right: calc(100% / 90);
+`;
+
+const imgContainer = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 100vh;
```

Comment:
```suggestion
display: flex;
  justify-content: space-between;
  align-items: center;

  width: 100%;
  height: 100vh;
  padding: 0 1.6rem;
```

요렇게 두개 바꾸면 이미지 사이즈 그대로 줄어드는데 위아래 공백이 쪼끔씩 생기긴 합니다..! 모달같은 경우는 제대로 물어보지 않아서 디자이너가 원하는 디자인이 무엇인지는 모르겠지만 개인적으로 이미지 크게보기하는 이유는 전체 이미지를 보기 위해서라고 생각해서 잘리거나 왜곡 없이 보이는게 더 좋을 것 같습니다!!

Reply:
움 개인적으로는 모달을 넘길 때마다 이미지 크기가 달라진다면 오히려 어색하게 느껴질 것 같아요...! 모달 상/하에 여백이 생기는 것도 기존 디자인과 너무 달라지는 느낌이구용 ... ㅠㅠ
그런데 또 도리 말대로 전체 이미지를 보려고 하는건데 일부가 잘린다면 그것도 ... 좋진 않은 듯 해서 일단 디자이너 분들께 질문 남겨뒀습니당 !!!

그래두 좋은 의견 고마워요 ♥︎♥︎